pkgs/tools/text/source-highlight/default.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, stdenv, fetchpatch, fetchurl, boost, updateAutotoolsGnuConfigScriptsHook }:
+{ lib, stdenv, fetchpatch, fetchurl, boost, updateAutotoolsGnuConfigScriptsHook, llvmPackages }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (rec {
```

Comment:
```suggestion
stdenv.mkDerivation (rec {
```

Redundant parenthesis?

Reply:
Seems like it, I removed the extra parentheses.